### ISSUE TITLE
Feature Requirement: I want to find out the unused files whose exports are all unused.

### ISSUE THREAD

**Issue by brandonxiang** [pzavolinsky/ts-unused-exports#253]
For example: 

The export like `c` is unused. However, `a` and `b` are used by another files

Right now, `ts-unused-exports` only can tell us `c`. This kind of file cannot be easily removed. But we want to know all the unused filenames exactly, and remove them by scripts efficiently.

```
export function c(x: number) {
  a(b);
  return x + 2;
}

// used by another files
export const a = (b: number) => {
  console.log(1);
};

// used by another files
export const b = 11;
```

Any thoughts about this?

**Comment by brandonxiang** [comment]
I want to add a method `getUnusedFiles`  in `app.ts`, to find out all unused files whose exports are all unused.

**Comment by mrseanryan** [comment]
Released as 9.0.0 - thank you
